data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+
+        {PETRIFY}
+    [/event]
+    [event]
+        name=moveto
+
+        [filter]
+            x,y=38,12
+            [not]
+                id=Dacyn
+            [/not]
+        [/filter]
+
+        {PETRIFY}
+    [/event]
+
+    # un-petrify anyone who got petrified
+    [event]
+        name=side 1 turn refresh
+        first_time_only=no
+        {MODIFY_UNIT side=2,3,4,5,6,7,8,9 status.petrified no}
+    [/event]
+    [event]
+        name=side 1 turn end
+        first_time_only=no
+        {MODIFY_UNIT (side=1) status.petrified no}
+    [/event]
+
+    # spawn vault defender
+    [event]
+        name=enter_hex
+        [filter]
+            side,x,y=1,43,12
+        [/filter]
+        [cancel_action]
+        [/cancel_action]
+        [message]
+            speaker=unit
+            message= _ "Hmm, looks like a magic lamp. I’ve heard stories that these can grant wishes. Today must be my lucky day!"
+        [/message]
+
+        {QUAKE cave-in.ogg}
+        [delay]
+            time=1500
+        [/delay]
+
+        [sound]
+            name=melee-fire.ogg
+        [/sound]
+
+        # the petrification traps are pretty good against this guy, if you can lure him onto them
+        {NAMED_UNIT 2 (Elder Jinn) 42 11 "Jinn" "" ()} {FACING se} {ANIMATE}
+        [object]
+            [filter]
+                type=Elder Jinn
+            [/filter]
+
+            [effect]
+                apply_to=hitpoints
+                heal_full,increase_total=yes,{ON_DIFFICULTY -45 -15 0}
+            [/effect]
+
+            [effect]
+                apply_to=alignment
+                set={ON_DIFFICULTY lawful neutral chaotic}
+            [/effect]
+        [/object]
+
+        [delay]
+            time=500
+        [/delay]
+
+        [message]
+            speaker=Jinn
+            message= _ "WHO DARES DISTURB MY SLUMBER?"
+        [/message]
+    [/event]
+
+    # worthless green cloak
+    [event]
+        name=moveto
+        [filter]
+            side,x,y=1,38,10
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "This fine silken cloak would be priceless back in Wesnoth, but is useless to us here."
+        [/message]
+    [/event]
+
+    # open vault chests
+    [event]
+        name=moveto
+
+        [filter]
+            side,x,y=1,43,10
+        [/filter]
+
+        {REMOVE_IMAGE 43 10}
+        {PLACE_IMAGE "items/chest-open.png~FL(horiz)" 43 10}
+        {VARIABLE gold 45}
+        [message]
+            speaker,sound=unit,gold.ogg
+            message= _ "This chest contains $gold gold!"
+        [/message]
+        [gold]
+            side,amount=1,$gold
+        [/gold]
+    [/event]
+    [event]
+        name=moveto
+
+        [filter]
+            side,x,y=1,42,9
+        [/filter]
+
+        {REMOVE_IMAGE 42 9}
+        {PLACE_IMAGE "items/chest-open.png~FL(horiz)" 42 9}
+        [message]
+            speaker=unit
+            message= _ "It’s full of... toilet paper?!?"
```

Comment:
Joke? No need for a second question mark, in my opinion.

Reply:
Do they even know what toilet paper is?

Reply:
Yeah, just intended as a joke.  If anybody has a period-appropriate substitute that would work fine as well.

Reply:
There's a recurring character in the Zelda series that's just a hand in a toilet asking for toilet paper, and they have a similar high-fantasy setting, so I think it works

Reply:
I'm not quite sure what the intention is, so I can't really suggest a period-appropriate substitute. I am assuming "period equivalent of toilet paper" is not the correct answer, as that would almost certainly fail to leave an equivalent impression. I'm not entirely sure what the period equivalent of toilet paper _would_ be, but it's probably not some disposable product like toilet paper.

Reply:
If it's a joke, I think anachronistic toilet paper is fine.